integration/setup/setup.go

```diff
@@ -98,7 +99,27 @@ func Resources(g *framework.Guest, h *framework.Host) error {
 			return microerror.Mask(err)
 		}
 
-		err = h.InstallBranchOperator("kvm-operator", "kvmconfig", template.KVMOperatorChartValues)
+		var values string
+		{
+			c := chartvalues.KVMOperatorConfig{
+				ClusterName: env.ClusterID(),
+				ClusterRole: chartvalues.KVMOperatorClusterRole{
+					BindingName: fmt.Sprintf("%s-kvm-operator", env.ClusterID()),
+					Name:        fmt.Sprintf("%s-kvm-operator", env.ClusterID()),
+				},
+				ClusterRolePSP: chartvalues.KVMOperatorClusterRole{
+					BindingName: fmt.Sprintf("%s-kvm-operator-psp", env.ClusterID()),
+					Name:        fmt.Sprintf("%s-kvm-operator-psp", env.ClusterID()),
```

Comment:
For the future it would be cool to put these things in the `env` package or lets just have a `key` package.

Reply:
I'll do that in a follow up with other operators, I think the env file makes sense for now.